draft-sharabayko-mops-srt.md

```diff
@@ -1249,6 +1239,25 @@ of the handshake Extension Message Flags to 1.
 In this mode consecutive packets form one continuous stream that can be read, with 
 portions of any size.
 
+### Use Case Live Streaming {#use-case-live-streaming}
+
+This section describes real world examples of live audio/video streaming and the current consensus, which is maintaining compatibility between SRT implementations by different vendors. It's meant as guidance for developers to be compatible with existing SRT implementations.
+
+The term "live streaming" refers to MPEG-TS style continuous data transmission with latency management. Live streaming based on segmentation and transmission of files like in HLS protocol is not part of this use case.
+
+The default mode for continuous live streaming is Message Mode. 
```

Comment:
There is also a need to say that in the case of live streaming live congestion control (LiveCC) is used and refer to an appropriate section. Should be "SRT Packet Pacing and Congestion Control".

> As SRT is designed both for live streaming and file/message transmission (Section 4.2), there are two groups of congestion control algorithms defined in SRT: live congestion control (LiveCC), and file transfer congestion control (FileCC).